src/main/java/edu/njit/jerse/ashe/Ashe.java

```diff
@@ -118,18 +119,22 @@ public static void run(String root, String targetFile, String targetMethod, Stri
                     LOGGER.info("Exiting...");
                     return;
                 }
-                LOGGER.error("Errors were found but not replaced with {} response.", model);
-                throw new RuntimeException("Errors were not replaced with " + model + " response.");
+
+                String errorMessage = "Errors were found but not replaced with " + model + " response.";
+                LOGGER.error(errorMessage);
+                throw new RuntimeException(errorMessage);
             }
             LOGGER.info("Errors replaced with {} response successfully.", model);
 
+
```

Comment:
In general, it is good to avoid whitespace changes in a pull request.  It's better to put them in their own pull request.